components/BackButton/index.jsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+import { useRouter } from 'next/router';
+
+import backButton from '@/public/back-button.svg';
```

Comment:
I see sometimes there is an empty line between local imports and public imports (like in this case) and in other files there is not this separation. 
 It could be interesting to choose 1 way and apply it everywhere — though I am fine with either way (with or without separations).
 
 cc @carryall if you have any recommendation (it seems we do not have anything in the compass yet) :)

Reply:
In Varun, [we group and ordered by `buildin`, `internal` and `external`](https://github.com/nimblehq/varun-web/blob/develop/.eslintrc.js#L45) followed [Airbnb ESLint](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/imports.js#L149) No sure we have that yet on nimble ESLine 🤔  I'll add an issue on Compass 🙌🏼

Reply:
separated import groups by spaces in all components in 38f4208b896b6b68a73b99e02c90bb17040037f8